aimmo-game-creator/service.py

```diff
@@ -7,9 +7,8 @@
 
 def main():
     logging.basicConfig(level=logging.DEBUG)
-    WorkerManagerClass = WORKER_MANAGERS[os.environ.get('WORKER_MANAGER', 'local')]
-    worker_manager = WorkerManagerClass(os.environ.get('GAME_API_URL',
-                                        'http://localhost:8000/players/api/games/'))
+    WorkerManagerClass = WORKER_MANAGERS[os.environ.get('WORKER_MANAGER', None)]
```

Comment:
It doesn't really make sense to replace either of these with a default of `None` - it'll just break anyway.

If we truly don't want the default, we should just use `os.environ['WORKER_MANAHER']`, so it breaks with a nice error if we're missing the environment variable